k6-operator/cluster/upstream/clusterRole.yaml

```diff
@@ -0,0 +1,241 @@
+# Source: k6-operator/templates/clusterRole.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: k6-operator-manager-role
```

Comment:
this is quite a powerful role, including reading all secrets, and full control over all the cluster deployments. Ideally we'll have this be a "role" instead of a "clusterrole" and bind it to opt-in namespaces

Reply:
this is the ClusterRole that comes out-of-the-box. is the suggestion to move this to the controller package and transform to role and for now deploy to dev-enablement? 

are there attributes between ClusterRole and Role transferable? wondering if it is a matter of just location and replacing values for `kind` and adding `.metadata.namespace`